test_accounts/appeals_status_test_accounts.md

```diff
@@ -0,0 +1,32 @@
+# Sandbox Test Accounts for Appeals Status API (v1)
+
+This document provides test account information for the [Appeals Status API (v1)](https://developer.va.gov/explore/api/appeals-status/docs?version=current). The API returns synthetic VA records for these test accounts in the sandbox environment.
+
+## Authentication and Authorization
+
+The authentication model for the [Appeals Status API (v1)](https://developer.va.gov/explore/api/appeals-status/docs?version=current) uses OAuth 2.0/OpenID Connect. The following authorization models are supported:
+- [Authorization code flow](https://developer.va.gov/explore/api/appeals-status/authorization-code)
+- [Client Credentials Grant (CCG)](https://developer.va.gov/explore/api/appeals-status/client-credentials)
+
+**To use this test data**
+1. Find the test accounts that meet your use case from the table below.
+2. Access login credentials for these accounts and instructions for using ID.me and Login.gov by following the link in the email that was sent to you when you signed up for sandbox access.  If you do not have this email, [request sandbox access](https://developer.va.gov/explore/api/appeals-status/sandbox-access) on developer.va.gov.
```

Comment:
This URL isn't live on the prod developer portal yet, but it will be when we make the API publicly available.